packages/aws-cdk/lib/context-providers/index.ts

```diff
@@ -69,6 +69,11 @@ export async function provideContextValues(
       }, resolvedEnvironment, sdk);
 
       value = await provider.getValue({ ...missingContext.props, lookupRoleArn: arns.lookupRoleArn });
+
+      if (missingContext.disableContextCaching) {
+        debug(`Skipping context caching for ${key}`);
+        context.set(key + TRANSIENT_CONTEXT_KEY, { [TRANSIENT_CONTEXT_KEY]: true });
```

Comment:
Does this property work for any context lookup? Like it was suggested in [this comment](https://github.com/aws/aws-cdk/issues/12366#issuecomment-755093975)

Reply:
It should be able to be used it in other context lookups, but this PR only implements it for SSM. I'm not sure what other context lookups would benefit from this, but with the underlaying changes in this PR, it would be easy to make it work elsewhere.